webpack.config.node.js

```diff
@@ -93,7 +93,7 @@ const webpackConfig = {
 	},
 	resolve: {
 		extensions: [ '', '.json', '.js', '.jsx' ],
-		root: [ path.join( __dirname, 'server' ), path.join( __dirname, 'client' ), __dirname ],
```

Comment:
Yeah, I guess even with the entry path, `__dirname` should probably still be first. I'll update it.